src/stylesheets/components/_hide-this-page.scss

```diff
@@ -0,0 +1,86 @@
+@media only screen {
+    .js-enabled .govuk-quick-exit__container.govuk-visually-hidden {
+      display:block;
+    }
+    .govuk-quick-exit__container.govuk-quick-exit__enabled {
+      display:block;
+      float: right;
+      position: -webkit-sticky;
+      position: sticky;
+      top:0;
+      z-index:1000;
+      .govuk-quick-exit__position {
+        z-index:1000;
+        background-color:govuk-colour("white");
+        .govuk-quick-exit {
+          width:100%;
+          margin-bottom:0;
+        }
+        .govuk-quick-exit--info {
+          text-align: center;
+          padding: 10px 0;
+          border: 1px $govuk-border-colour solid;
+          border-top-style: none;
+          margin: 0;
+  
+        }
+      }
+      .govuk-quick-exit__IEsizing{
+        visibility:hidden;
+      }
+    }
+  }
+  .govuk-quick-exit__position {
+    position:fixed;
+  }
+  @media only screen and (min-width: 641px) {
+    @supports (position:sticky) {
+      .govuk-quick-exit__position {
+        position:sticky;
+        top:0;
+      }
+    }
+  }
+  
+  @media only screen and (max-width: 640px) {
+    .govuk-quick-exit__IEsizing {
+      display:none;
+    }
+  
+    .js-enabled .govuk-quick-exit__container.govuk-visually-hidden {
+      display:block;
+  
+      ~.gem-c-cookie-banner__confirmation,.gem-c-cookie-banner {
```

Comment:
It feels odd that there's references to (what I assume is) the GOV.UK Publishing Components gem's cookie banner component in this. I can see why it'd be useful, but only for apps that use the gem - and for everyone else it's (an admittedly tiny) bit of extra code that isn't useful. 

I think that a component's styles should be isolated in their effect - and this breaks that isolation by affecting other components.

Reply:
Hey Ian! I'll get Malcolm, our dev, to respond to this when he gets back from leave.